sidebars.ts

```diff
@@ -340,6 +340,15 @@ module.exports = {
                 'send-data/hosted-collectors/http-source/troubleshooting',
               ],
             },
+            {
+              type: 'category',
+              label: 'Webhooks Sources',
+              collapsible: true,
+              collapsed: true,
+              items: [
+                'send-data/hosted-collectors/zoom',
```

Comment:
@JV0812 Are you creating a new Hosted Collector subcategory called Webhook Sources? If so, you'll need to change the link to `send-data/hosted-collectors/webhook-sources/zoom` and place the doc in the correct place.

Reply:
Okay, I will confirm with Megha on this.

Reply:
@JV0812 I've fixed the link here, created a webhook-sources folder, and dropped the Zoom.md file there. Can you take a look?

Reply:
@kimsauce I got the go ahead from Arpit Jaiswal on the location of this doc, he is fine with the location where the doc is currently present.